src/ui/component/collapsible/collapsible.tsx

```diff
@@ -0,0 +1,52 @@
+import type { FC } from 'react'
+import { useState, useCallback } from 'react'
+import classNames from 'classnames'
+import * as RCollapsible from '@radix-ui/react-collapsible'
+import type { IconName } from '@/ui/component/icon/icon'
+import { Icon } from '@/ui/component/icon/icon'
+import type { AssertSubset } from '@/util/type'
+import './collapsible.scss'
+
+type CollapsibleProps = {
+  children: JSX.Element
```

Comment:
I prefer the term `content` to `children`, which is what Radix uses. It seems more meaningful to me, imho.

Reply:
`children` is a conventional term in React, but if `content` provides more clarity we can update the props.